src/utils/email.js

```diff
@@ -58,23 +63,23 @@ function sendEmail(options, data) {
 /**
  * Sends an email template to the specified user
  */
-function sendTemplateTo(templateName, user, options, data) {
+function sendTemplateTo(templateName, user, email, options, data) {
```

Comment:
Ça serait pas plus propre de gérer ce cas spécial pas mal unique en passant `to` dans les options? C'est pour ce genre de cas que ce paramètre existe et est passé directement à `sendEmail` par la suite.

L'idée derrière cette fonction c'est d'abstraire la gestion de l'adresse courriel, et d'envoyer le courriel à un "utilisateur" plutôt qu'à une adresse.

Reply:
effectivement beaucoup plus propre! j'avais pas vu les options dans `sendEmail`